docs/examples/modes.py

```diff
@@ -0,0 +1,190 @@
+    lambda x: x > 0
+).dropna().unstack("spores")
+
+# %% [markdown]
+# ## Visualising results
+#
+# We can use [plotly](https://plotly.com/) to quickly examine our results.
+# These are just some examples of how to visualise Calliope data.
+
+# %%
+# We set the color mapping to use in all our plots by extracting the colors defined in the technology definitions of our model.
+# We also create some reusable plotting functions.
+colors = model_plan.inputs.color.to_series().to_dict()
+
+
+def plot_flows(results: xr.Dataset) -> go.Figure:
+    df_electricity = (
+        (results.flow_out.fillna(0) - results.flow_in.fillna(0))
+        .sel(carriers="power")
+        .sum("nodes")
+        .to_series()
+        .where(lambda x: x != 0)
+        .dropna()
+        .to_frame("Flow in/out (kWh)")
+        .reset_index()
+    )
+    df_electricity_demand = df_electricity[df_electricity.techs == "demand_power"]
+    df_electricity_other = df_electricity[df_electricity.techs != "demand_power"]
+
+    fig = px.bar(
+        df_electricity_other,
+        x="timesteps",
+        y="Flow in/out (kWh)",
+        color="techs",
+        color_discrete_map=colors,
+    )
+    fig.add_scatter(
+        x=df_electricity_demand.timesteps,
+        y=-1 * df_electricity_demand["Flow in/out (kWh)"],
+        marker_color="black",
+        name="demand",
+    )
+    return fig
+
+
+def plot_capacity(results: xr.Dataset, **plotly_kwargs) -> go.Figure:
+    df_capacity = (
+        results.flow_cap.where(results.techs != "demand_power")
+        .sel(carriers="power")
+        .to_series()
+        .where(lambda x: x != 0)
+        .dropna()
+        .to_frame("Flow capacity (kW)")
+        .reset_index()
+    )
+
+    fig = px.bar(
+        df_capacity,
+        x="nodes",
+        y="Flow capacity (kW)",
+        color="techs",
+        color_discrete_map=colors,
+        **plotly_kwargs,
+    )
+    return fig
+
+
+# %% [markdown]
+# ## `plan` vs `operate`
+# Here, we compare flows over the 10 days.
+# Note how flows do not match as the rolling horizon makes it difficult to make the correct storage charge/discharge decisions.
+
+# %%
+fig_flows_plan = plot_flows(
+    model_plan.results.sel(timesteps=model_operate.results.timesteps)
+)
+fig_flows_plan.update_layout(title="Plan mode flows")
+
+
+# %%
+fig_flows_operate = plot_flows(model_operate.results)
+fig_flows_operate.update_layout(title="Operate mode flows")
+
+# %% [markdown]
+# ## `plan` vs `spores`
+# Here, we compare installed capacities between the baseline run (== `plan` mode) and the SPORES.
+# Note how the baseline SPORE is the same as `plan` mode and then results deviate considerably.
+
+# %%
+fig_flows_plan = plot_capacity(model_plan.results)
+fig_flows_plan.update_layout(title="Plan mode capacities")
+
+# %%
+fig_flows_spores = plot_capacity(model_spores.results, facet_col="spores")
+fig_flows_spores.update_layout(title="SPORES mode capacities")
```

Comment:
Where is this shown in the published docs? I cannot find it.

Reply:
Ok, but this part was completely detached from the SPORES docs, making it unlikely to find when looking at SPORES. I've made a commit to facilitate the connection.